### ISSUE TITLE
header layout: chrome/firefox desync

### ISSUE THREAD

**Issue by nev-r** [DestinyItemManager/DIM#5621]
tldr header looks bad in firefox, the text elements are too tall. css seems designed around, maybe some kind of chrome flex quirk?

![image](https://user-images.githubusercontent.com/68782081/90766843-0d5c8280-e2a1-11ea-8610-c9965b346996.png)
![image](https://user-images.githubusercontent.com/68782081/90766956-3715a980-e2a1-11ea-81bd-f8a69744f6cf.png)
![image](https://user-images.githubusercontent.com/68782081/90767213-aab7b680-e2a1-11ea-8b3f-c5c67c1c2bc3.png)
![image](https://user-images.githubusercontent.com/68782081/90766997-47c61f80-e2a1-11ea-9099-419b20e3d20a.png)
firefox (correctly?) fits the `.class` inside the `.top`,
and the `.top` inside the `.character-text`
and puts the 4px margin around `.character-text`

.

![image](https://user-images.githubusercontent.com/68782081/90768106-15b5bd00-e2a3-11ea-9916-a861fdebc4c6.png)
![image](https://user-images.githubusercontent.com/68782081/90767403-04b87c00-e2a2-11ea-9594-5fc767c90fb0.png)
![image](https://user-images.githubusercontent.com/68782081/90767425-113cd480-e2a2-11ea-968d-a898145776db.png)

chrome somehow makes `.class` bigger than its container (`.top`)
and overflows it upward and downward, as if it has -2px margins
